docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -530,593 +325,696 @@ ALTER TABLE ONLY public.users
+
+CREATE INDEX idx_json ON public.test_json (data);
 ```
 
 ---
 
-### Exporting data with names for tables/functions/procedures using special characters/whitespaces fails
+### GIN indexes on multiple columns are not supported
 
-**GitHub**: [Issue #636](https://github.com/yugabyte/yb-voyager/issues/636), [Issue #688](https://github.com/yugabyte/yb-voyager/issues/688), [Issue #702](https://github.com/yugabyte/yb-voyager/issues/702)
+**GitHub**: [Issue #724](https://github.com/yugabyte/yb-voyager/issues/724)
 
-**Description**: If you define complex names for your source database tables/functions/procedures using backticks or double quotes for example, \`abc xyz\` , \`abc@xyz\`, or "abc@123", the migration hangs during the export data step.
+**Description**: If there are GIN indexes in the source schema on multiple columns, they result in an error during import schema as follows:
 
-**Workaround**: Rename the objects (tables/functions/procedures) on the source database to a name without special characters.
+```output
+ERROR: access method "ybgin" does not support multicolumn indexes (SQLSTATE 0A000)
+```
+
+**Workaround**: Currently, as there is no workaround, modify the schema to not include such indexes.
 
 **Example**
 
-An example schema on the source MySQL database is as follows:
+An example schema on the source database is as follows:
 
 ```sql
-CREATE TABLE `xyz abc`(id int);
-INSERT INTO `xyz abc` VALUES(1);
-INSERT INTO `xyz abc` VALUES(2);
-INSERT INTO `xyz abc` VALUES(3);
+CREATE TABLE public.test_gin_json (
+    id     integer,
+    text   jsonb,
+    text1  jsonb
+);
+
+CREATE INDEX gin_multi_on_json
+    ON public.test_gin_json USING gin (text, text1);
 ```
 
-The exported schema is as follows:
+---
+
+## Concurrency control
+
+### Advisory locks is not yet implemented
+
+**GitHub**: [Issue #3642](https://github.com/yugabyte/yugabyte-db/issues/3642)
+
+**Description**: YugabyteDB does not support PostgreSQL advisory locks (for example, pg_advisory_lock, pg_try_advisory_lock). Any attempt to use advisory locks will result in a "function-not-implemented" error as per the following example:
 
 ```sql
-CREATE TABLE "xyz abc" (id bigint);
+yugabyte=# SELECT pg_advisory_lock(100), COUNT(*) FROM cars;
 ```
 
-The preceding example may hang or result in an error.
+```output
+ERROR:  advisory locks feature is currently in preview
+HINT:  To enable this preview feature, set the GFlag ysql_yb_enable_advisory_locks to true and add it to the list of allowed preview flags i.e. GFlag allowed_preview_flags_csv. If the app doesn't need strict functionality, this error can be silenced by using the GFlag yb_silence_advisory_locks_not_supported_error. See https://github.com/yugabyte/yugabyte-db/issues/3642 for details
+```
+
+**Workaround**: Implement a custom locking mechanism in the application to coordinate actions without relying on database-level advisory locks.
 
 ---
 
-### Importing with case-sensitive schema names
+### Events Listen / Notify
```

Comment:
Lets move this one to `Server Programming` as we discussed in the doc. cc: @makalaaneesh

Reply:
Moved! Thanks for review!